client/src/components/App.jsx

```diff
@@ -1,47 +1,39 @@
+/* eslint-disable no-console */
 /* eslint-disable no-undef */
 /* eslint-disable no-unused-vars */
 import React from 'react';
-import Slide from './Slide.jsx';
-import LeftArrow from './LeftArrow.jsx';
-import RightArrow from './RightArrow.jsx';
+import axios from 'axios';
 import ThumbnailGallery from './ThumbnailGallery.jsx';
+import Modal from './Modal.jsx';
+import '../../dist/styles.css';
 
-const sliderStyle = {
-  height: '520px',
-  width: '900px',
-  position: 'relative',
-  margin: '0 auto',
-  overflow: 'hidden',
-  whiteSpace: 'nowrap',
-};
-
-class Slider extends React.Component {
+class App extends React.Component {
   constructor(props) {
     super(props);
 
     this.state = {
-      images: [
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample26.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample14.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample43.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample22.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample8.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample17.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample33.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample19.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample2.jpg',
-        'https://s3-us-west-1.amazonaws.com/hackreactorfrontendcapstone/hrsf114-front-end-capstone-pics/sample5.jpg',
-      ],
+      images: [],
       currentIndex: 0,
-      translateValue: 0,
+      modal: false,
     };
   }
 
+  componentDidMount = () => {
+    const Id = window.location.pathname.split('/')[2];
+    axios.get(`/api/hostels/${Id}`)
+      .then(response => (
+        // console.log(response.data[0].photos)
+        this.setState({
```

Comment:
Remove the commented out console log and it looks good to go!

Reply:
ok!